src/utils.js

```diff
@@ -151,6 +174,9 @@ export const setVerboseMode = () => (process.env['LUMIGO_VERBOSE'] = 'TRUE');
 
 export const setSwitchOff = () => (process.env['LUMIGO_SWITCH_OFF'] = 'TRUE');
 
+export const setValidAliases = aliases =>
```

Comment:
I can see it's used just for testing, can't you change the value of the environment variable before the test?